src/css/index.scss

```diff
@@ -1,2 +1,356 @@
-@import './global';
-@import './information';
+$primary_color: #ffffff;
+$secondary_color: #070a0e;
+$tertiary_color: #94a3ba;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+.bgimg {
+  background: url('../css/cv.jpg') fixed no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  width: 100%;
+  padding: 20px 0 0 0;
+  height: 100px;
+  position: fixed;
+  z-index: 1;
+}
+
+.headleft {
+  width: 50%;
+  float: left;
+}
+
+.headright {
+  width: 50%;
+  float: right;
+}
+
+#home {
+  text-decoration: none;
+  font-size: 28px;
+  color: $primary_color;
+  font-weight: bold;
+  float: left;
+  margin-left: 10%;
+}
+
+#download {
+  margin-right: 10%;
+  float: right;
+  text-decoration: none;
+  border: 2px solid $primary_color;
+  font-size: 18px;
+  padding: 4px 25px;
+  color: $primary_color;
+  &:hover {
+    background: $btn_hover_color;
+  }
+}
+
+.skills-section {
+  position: relative;
+  width: 100%;
+  h2 {
+    color: $primary_color;
+    letter-spacing: 4px;
+    max-width: 949px;
+    margin: 0 auto;
+    &::after {
+      content: '';
+      display: block;
+      width: 100px;
+      border: 3px solid $btn_hover_color;
+      margin: 30px 0;
+    }
+  }
+}
+
+.skill-inner-width {
+  width: 949px;
+  margin: 0 auto 20px;
+  display: flex;
+}
+
+.skill-left,
+.skill-right {
+  height: 436px;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.skill-left {
+  width: 513px;
+}
+.skill-right {
+  width: 436px;
+}
+
+.skill {
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+}
+
+.skill_name,
+.skill_level {
+  flex: 1;
+}
+
+.skill_level {
+  font-weight: bold;
+}
+
+#msexcel,
+#msword {
+  width: 147px;
+  height: 147px;
+  order: 0;
+  .skill_name {
+    font-size: 10px;
+    transform: translateY(115%);
+  }
+  .skill_level {
+    font-size: 18px;
+    transform: translateY(-20%);
+  }
+}
+
+#adobe,
+#french {
+  width: 219px;
+  height: 219px;
+  .skill_name {
+    font-size: 15px;
+    font-weight: 500;
+    transform: translateY(115%);
```

Comment:
I think you make things too complex here with transform.
should be something really simple like:
```
.skill {
    display: flex;
    flex-direction: column;
    justify-content: center;
    text-align: center;
}
.skill_name, .skill_level {
  margin: 0;
}
```